lib/Conversion/CMakeLists.txt

```diff
@@ -14,8 +14,8 @@ XTenToLinalg.cpp
 DEPENDS
 XTenConversionIncGen
 XTenDialect
-ATenToXTenIncGen
-ATenToXTenIncGen2
+# ATenToXTenIncGen
```

Comment:
Please also provide a comment for this, or delete it if the dependencies will never be used in this setup.